src/core/core.scale.js

```diff
@@ -681,16 +681,16 @@ module.exports = Element.extend({
 		var me = this;
 		var chart = me.chart;
 		var display = me.options.display;
-		var i, ilen, meta;
```

Comment:
Can you revert these changes, it's done like that to actually make the final build smaller.